### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by humaton**
Add information to openshift README

### CODE REVIEW

File: openshift/README.md [hunk 1]
```diff
@@ -20,4 +20,11 @@ Use generated password to update RDS_PASSWORD value
...
 
-`$./deploy.sh`
+`$./deploy.sh
+
+## Deploy your changes to dev-cluster
+To deploy your changes, open PR from your fork to upstream repository. 
+CentosCI will build you image and comment on the PR with image name and tag.
+Your image is available in the registry: docker pull registry.devshift.net/fabric8-analytics/worker-scaler:SNAPSHOT-PR-25
+
+To update you dev deployment use [oc edit](https://docs.openshift.com/container-platform/3.4/cli_reference/basic_cli_operations.html#edit) from commandline or editor in web interface.
```
Comment (tisnik) [inline openshift/README.md]: update you->update your?

File: openshift/README.md [hunk 2]
```diff
@@ -20,4 +20,11 @@ Use generated password to update RDS_PASSWORD value
 ## Deploy fabric8-analytics services
 Just run the deploy script and enjoy!
 
-`$./deploy.sh`
+`$./deploy.sh
+
+## Deploy your changes to dev-cluster
+To deploy your changes, open PR from your fork to upstream repository.
```
Comment (tisnik) [inline openshift/README.md]: open _a_ PR from your fork to _the_ upstream repository

File: openshift/README.md [hunk 3]
```diff
@@ -20,4 +20,11 @@ Use generated password to update RDS_PASSWORD value
 ## Deploy fabric8-analytics services
 Just run the deploy script and enjoy!
 
-`$./deploy.sh`
+`$./deploy.sh
+
+## Deploy your changes to dev-cluster
+To deploy your changes, open PR from your fork to upstream repository. 
+CentosCI will build you image and comment on the PR with image name and tag.
```
Comment (tisnik) [inline openshift/README.md]: your image
with an image name

File: openshift/README.md (hunk 1 above)
Comment (tisnik) [reply openshift/README.md]: from _the_ command<space>line

File: openshift/README.md [hunk 4]
```diff
@@ -20,4 +20,11 @@ Use generated password to update RDS_PASSWORD value
 ## Deploy fabric8-analytics services
 Just run the deploy script and enjoy!
 
-`$./deploy.sh`
+`$./deploy.sh
+
+## Deploy your changes to dev-cluster
+To deploy your changes, open PR from your fork to the upstream repository.
```
Comment (msrb) [inline openshift/README.md]: As @tisnik pointed out: open __a__ PR

**Pull request merged**